src/components/Queries.js

```diff
@@ -11,4 +11,28 @@ export const GET_USER = gql`
         thirtydayAverageShowerTime
       }
     }
-  `;
\ No newline at end of file
+  `;
+
+  export const SUBMIT_RECORD = gql`
+    mutation createDailyRecord(
+      $date: String!
+      $userId: Int!
+      $bagCount: Int!
+      $containerCount: Int!
+      $strawCount: Int!
+      $showerTime: Int!
+    ) {
+      createDailyRecord(input: {
+        date: $date
+        userId: $userId
+        bagCount: $bagCount
+        containerCount: $containerCount
+        strawCount: $strawCount
+        showerTime: $showerTime
+      }) {
+        containerCount
+        date
+        showerTime
```

Comment:
This looks great! Maybe we could move all mutations to a separate mutations file!